resources/views/settings/general.blade.php

```diff
@@ -163,6 +163,21 @@
                            </div>
                        </div>
 
+                       <!-- Shortcuts enable -->
+                       <div class="form-group {{ $errors->has('shortcuts_enabled') ? 'error' : '' }}">
+                           <div class="col-md-3">
+                               <strong> {{ trans('admin/settings/general.shortcuts_enabled') }}</strong>
+                           </div>
+                           <div class="col-md-9">
+                               <label class="form-control">
+                                   {{ Form::checkbox('shortcuts_enabled', '1', old('shortcuts_enabled', $setting->shortcuts_enabled)) }}
```

Comment:
Can we use an old-school checkbox here? Untangling ourselves from the form helpers for laravel 11 is going to be awful enough.